index.html

```diff
@@ -3013,9 +3013,20 @@ <h3>Algorithm</h3>
                   <var>nested value</var> expands to <code>@value</code>, an
                   <a data-link-for="JsonLdErrorCode">invalid @nest value</a> error
                   has been detected and processing is aborted.</li>
-                <li>Recursively repeat steps <a href="#alg-expand-each-key-value">13</a>
+                <li>Recursively repeat steps
+                  <a href="#alg-expand-property-scoped-context">3</a>,
+                  <a href="#alg-expand-property-scoped-context2">8</a>,
```

Comment:
why not also include step 9 (processing local context)?
I know that @niklasl didn't explicitly required it in #380,
but since we are there, not supporting local contexts seems as arbitrary as not supporting scoped contexts...

Reply:
The Syntax document is explicit where `@context` can be found. For instance, not in a value object or list object. It's really only in node objects and graph objects, so adding this would require a change to that, which is outside our scope.

Reply:
Got it :)